tools/ReleaseProcess/UploadArtifactsToGitHub.ps1

```diff
@@ -0,0 +1,444 @@
+        }
+
+        # Each artifact is a directory, fetch the packages within it.
+        $packages = $(Get-ChildItem -Path $artifactPath/* -Recurse `
+            -Include "*.deb", "*.rpm" `
+            -Exclude "*.src*", "*dev*", "*dbg*", "*debug*" `
+            | where { ! $_.PSIsContainer })
+
+        # Within each directory, rename the artifacts
+        $component,$os,$suffix = $artifactName.split('-')
+
+        # CentOs7 packages do not need renaming.
+        if ($os -eq "centos7")
+        {
+            echo "Skip renaming :"
+            echo $($packages.FullName)
+            $artifactFinalists += $packages;
+            continue;
+        }
+
+        # Ranaming the artifacts
+        foreach ($package in $packages)
+        {
+            echo "Processing : $($package.FullName)"
+            $name,$version,$arch,$suffix = $package.Name.split("_")
+            $arch,$ext,$suffix = $arch.split(".")
+
+            # Reconstruct the new name from the segments. 
+            $finalName = @($name, $version, $os, $arch) -join '_'
+            $finalName = "$finalName.$ext"
+            $newPath = $(Join-Path -Path "$($package.Directory)" -ChildPath "$finalName")
+
+            # Rename
+            Rename-Item -Path $package.FullName -NewName $newPath
+
+            # Record renamed files
+            $artifactFinalists += $(Get-Item $newPath)
+        }
+
+        echo ""
+    }
+
+    echo ""
+
+    # Stage uploading files
+    foreach ($artifact in $artifactFinalists)
+    {
+        echo "Moving : $($artifact.FullName)"
+        echo "To : $(Join-Path -Path $outputDir -ChildPath $artifact.Name)"
+        Move-Item -Path $artifact.FullName -Destination $(Join-Path -Path $outputDir -ChildPath $artifact.Name) -Force
+    }
+}
+
+
+<#
+    .SYNOPSIS
+        This function downloads and prepares the IoT Identity Service artifacts from Github Action for a release.
+        The final artifacts are collected under "<WorkDir>/output" path.
+    .NOTES
+        Requires Azure CLI to be login
+    .DESCRIPTION
+        1) Download edgelet artifacts from a given commit off of the Github build action:
+              https://github.com/Azure/iot-identity-service/actions/workflows/packages.yaml
+        2) Extract the *.zip artifacts into their own directories
+        3) Traverse the directories and rename a relavant artifact into the following format: 
+              .DEB _________________ {component}_{version}_{os}_{arch}.deb 
+              .RPM _________________ {component}-{version}.{os}.{arch}.rpm  (No name change)
+        4) Move the renamed artifacts to a staging directory at "<WorkDir>/output"
+    .PARAMETER CommitId
+        Github SHA256 commit id
+    .PARAMETER WorkDir
+        Absolute path of the working directory
+#>
+function Prepare-GitHub-Artifacts
+{
+    [CmdletBinding()]
+    param (
+        <# 
+        Github SHA256 commit id
```

Comment:
Pretty sure a git commit id is SHA1, not SHA256.

Reply:
I think you're right! We may want to migrate our repo to SHA256 from SHA1 if that's the case:
https://git-scm.com/docs/hash-function-transition/